src/desktop/apps/artwork/components/artist_artworks/index.jade

```diff
@@ -8,12 +8,22 @@ if artwork.artist && artwork.artist.artworks && artwork.artist.artworks.length
         href='#{artwork.artist.href}/works'
       ) View All
 
-    - var columns = helpers.artist_artworks.masonry(artwork.artist.artworks).columns
-    include ../../../../components/artwork_masonry/index
+    if sd.ENABLE_EXPERIMENTAL_STITCH_INJECTION
+      .artwork-masonry
+        #stitch-artwork-artist-artworks-masonry
+          -
+            const artworks = {
+              edges: artwork.artist.artworks.map(artwork => {
+                return {
+                  node: artwork
+                }
+              })
+            }
```

Comment:
Need to update Reaction to handle cases where an array of artworks is passed in, not a relay-connection. Once that's done these transforms will go away.